config/services.yml

```
@@ -30,6 +30,16 @@ earthdata:
     uvg_root: "https://uvg.sit.earthdata.nasa.gov"
     dmmt_root: "https://draftmmt.sit.earthdata.nasa.gov"
     gkr_root: "https://gkr.sit.earthdatacloud.nasa.gov"
+  dit:
+    urs_root: "https://uat.urs.earthdata.nasa.gov"
+    echo_root: "https://cmr.uat.earthdata.nasa.gov/legacy-services"
```

```suggestion
    echo_root: "https://cmr.maap-project.org/legacy-services"
```

Eventually this should be "cmr.dit.maap-project.org" but that instance of CMR is not working at the moment.